conf/messages

```diff
@@ -6,6 +6,9 @@ common.continue=Continue
 common.feedback=This is a new service - your {0} will help us to improve it.
 common.feedback.link=feedback
 
+common.navigation.messages = Messages
```

Comment:
`Messages` or `Conversations` ? It seems our dictionary is not always consistent. Sometime we use _message_ and other _conversation_.

Reply:
This is taken from the prototype. I am not sure if Conversations and Messages are the same concepts.